docs/source/deployment/gcp-functions.mdx

```diff
@@ -178,81 +177,6 @@ const server = new ApolloServer({
 exports.handler = server.createHandler();
 ```
 
-## Modifying the GCF Response (CORS)
-
-To enable CORS, you need to modify your HTTP response headers. To do so, use the `cors` option:
-
-```javascript{23-26}
-const { ApolloServer, gql } = require('apollo-server-cloud-functions');
-
-// Construct a schema, using GraphQL schema language
-const typeDefs = gql`
-  type Query {
-    hello: String
-  }
-`;
-
-// Provide resolver functions for your schema fields
-const resolvers = {
-  Query: {
-    hello: () => 'Hello world!',
-  },
-};
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-});
-
-exports.handler = server.createHandler({
-  cors: {
-    origin: '*',
-    credentials: true,
-  },
-});
-```
-
-To enable CORS response for requests with credentials (cookies, HTTP authentication) the allow origin header must equal the request origin and the allow credential header must be set to `true`.
-
-```javascript
-const { ApolloServer, gql } = require('apollo-server-cloud-functions');
-
-// Construct a schema, using GraphQL schema language
-const typeDefs = gql`
-  type Query {
-    hello: String
-  }
-`;
-
-// Provide resolver functions for your schema fields
-const resolvers = {
-  Query: {
-    hello: () => 'Hello world!',
-  },
-};
-
-const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-});
-
-exports.handler = server.createHandler({
-  cors: {
-    origin: true,
-    credentials: true,
-  },
-});
-```
-
-### CORS Options
-
-The options correspond to the [express `cors` configuration](https://github.com/expressjs/cors#configuration-options) with the following fields(all are optional):
+## Enabling production mode
 
-| Option | Type(s) |
-|--------|---------|
-| `origin` | boolean \| string \| string[] |
-| `methods` | string \| string[] |
-| `allowedHeaders` | string \| string[] |
-| `exposedHeaders` | string \| string[] |
-| `credentials` | boolean |
-| `maxAge` | number |
+When you consider your app to be production-ready, you should remember to edit it to remove the `NODE_APP` environment variable you set at the beginning. Among other things, this will change its landing page to be more discreet, and will mask error details from the end user.
```

Comment:
```suggestion
When you consider your app to be production-ready, you should remember to edit it to remove the `NODE_ENV` environment variable you set at the beginning. Among other things, this will change its landing page to be more discreet, and will mask error details from the end user.
```